ethpm_types/contract_type.py

```diff
@@ -469,6 +478,37 @@ def structs(self) -> ABIList:
         """
         return self._get_abis(filter_fn=lambda a: isinstance(a, StructABI))
 
+    @property
+    def natspecs(self) -> dict[str, str]:
+        """
+        A mapping of ABI selectors to their natspec documentation.
+        """
+        return {
+            **self._method_natspecs,
+            **self._event_natspecs,
+            **self._error_natspecs,
+            **self._struct_natspecs,
+        }
+
+    @cached_property
+    def _method_natspecs(self) -> Dict[str, str]:
+        return _extract_natspec(self.devdoc or {}, "methods", self.methods)
+
+    @cached_property
+    def _event_natspecs(self) -> Dict[str, str]:
+        # NOTE: I have only seen these appear on Solidity (not Vyper).
```

Comment:
note: Vyper doesn't allow docstrings for event defs (but that's a good idea)

Reply:
I see snekmate does it anyway! They just don't show up in the devdoc

Reply:
Yes, there's an issue tracking